docs/testplan/token-payment-processor-testplan.md

```diff
@@ -0,0 +1,31 @@
+|               | Minimum Confirmation                 | A transaction should not be called paid unless it meets the minimum number of confirmations reached (this should be visible in UI, pending until it meets minimum number of confirmations)                                                                                                               | Must have early on (Alpha)                                                                                                                             |
+|               | Cancel Transaction- 0 Confirmations  | User has a pending transaction and it has 0 confirmations, the user can cancel the transaction from their wallet by overriding it with a higher transaction fee with a different destination (change should be visible on UI, can show as either canceled or entirely dropped in UI)                     | Must have early on (Beta)                                                                                                                              |
+|               | Transaction Fee Warning              | A general warning message should be shown in the UI regarding low transaction fees and explain to users that if the fee is too low then there is a possibility that the transaction will be reverted (their gas fee won't be given back) or delayed                                                      | Nice to have                                                                                                                                           |
+|               | Multiple Transactions                | If a user has a pending transaction and then performs another transaction with a higher nonce using the same address, the new transaction has to wait until the previous transaction with the lower nonce is confirmed (standard behavior of geth, nothing to test for us)                               | Nothing to test. Keeping it in for documentation.                                                                                                      |
+| Alerts        | Resynced Geth Node                   | Geth nodes should be able to sync with the network, if a geth node resyncs it can take several days and so the token payment processor should pick up transactions during this resync. It should apply the USD value by the time the transaction hits the blockchain and not by the time we proccess it. | Must have early on (Beta)                                                                                                                              |
```

Comment:
We are determining the USD value of the token via blocktime not via the time the transaction is added to our DB so we should be covered on this.